onitama/src/cards.rs

```diff
@@ -379,19 +212,11 @@ mod tests {
     #[test]
     fn test_reverse_bitmap() {
         assert_eq!(
-            reverse_bitmap(Card::Eel.get_moves()),
-            Card::Cobra.get_moves()
+            Card::Eel.get_white().into_value(),
+            Card::Cobra.get_black().into_value()
         )
```

Comment:
This doesn't seem to use reverse_bitmap anymore, and also seems like it would always be wrong.

Reply:
the black cards a created by reversing the white cards, so this checks the precomputed result